admin_guide/metrics-sizing-guidelines.adoc

```diff
@@ -0,0 +1,164 @@
+of view. Openshift administrators need a way to monitor pods of interest and see how they perform
+ and react if necessary to fix critical issue, at other side Openshift users also need simple and
+ comprehensive overview of pods where their application run.
+
+This guide will provide Openshift metrics guidlines which Openshift administrators and users can
+follow when trying to dimension openshift metrics for openshift installations
+
+== Openshift Metrics Installation
+
+Installation of Openshift metrics is described in https://docs.openshift.com/enterprise/3.2/install_config/cluster_metrics.html[Openshift Enterprise v3 documentation]
+For instruction how to configure Opensift Enterprise Metrics on functional Openshift cluster, please follow instruction
+in https://docs.openshift.com/enterprise/3.2/install_config/cluster_metrics.html[Openshift Enterprise documentation]
+
+However, it is very important to fulfill below points when starting Opensift metrics in order to avoid potential issues which can appear later during deployment
+
+
+- It is important to use `persistant_storage=true` parameter. Persistent storage parameter is necessary if we want metrics
+data to persist across cassandra metrics pod lifetime.
+Prior using `persistant_storage=true` it is necessaary to have available an peristant volume which can be used by metrics cassandra pod.
+Please check https://docs.openshift.com/enterprise/3.2/dev_guide/persistent_volumes.html[Openshift Enterprise documentation - how to create persistant volume]
+for detailed instructions how to create Persistent Volume ( PV )
+
+Openshift metrics supports also https://github.com/openshift/origin-metrics/blob/master/metrics.yaml#L130[dynamically provisioned persistent volumes]
+Persistent Volumes ( PV ) and to use this feature with Openshift metrics, it is necessary for Openshift metrics to be started with `
+DYNAMICALLY_PROVISION_STORAGE=true` parameter. At time only EBS, GCE and Cinder storage backends can be used in order to dynamically
+provision persistent volumes for Openshift. For more inforamtion related to dynamiclly provisioned Persistent Volumes, please check
+https://docs.openshift.com/enterprise/3.2/install_config/persistent_storage/dynamically_provisioning_pvs.html[Dynamically Provisioned PVS]
+
+
+If `persistent_storage=true` parameter is specified, then Openshift metrics will create PVC and use it
+as storage space for cassandra pod withing Openshift metrics. Important to note, that all what applies for cassandra database performances for case it
+it is installed out of Openshift metrics, also applies on cassandra when deployed as part of Openshift Metrics. For detailed instructions regarding cassandra databases
+is necessary to check official http://cassandra.apache.org/doc/latest/[cassandra documentation]
+
+
+- If `persistent_storage=false` is used and with intention for metrics data not to live longer than pod lifetime,
```

Comment:
s/is used/is used, metrics data is ephemeral and will be lost once the pod is stopped.